src/Chapter1.hs

```diff
@@ -187,6 +187,8 @@ ghci> :q
 
 -}
 
+import Data.List
```

Comment:
As we have tests in the above comments as well, the compiler expects all imports go before any functions and code 😞  So, in order to make it work, this line should go after the `module Chapter1 where` 🙂 
We are talking a bit more about imports in the next Chapter, sorry for the inconvenience.

Generally, there should not be the need in any additional imports, but I think that something exciting in your solutions is awaiting 👍🏼 😊